custom_components/waste_collection_schedule/waste_collection_schedule/source/cidiu_it.py

```diff
@@ -0,0 +1,171 @@
+import logging
+from datetime import datetime
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "CIDIU S.p.A."
+DESCRIPTION = "Source for CIDIU waste collection services for the north-west Turin province"
+URL = "https://cidiu.it/"
+
+TEST_CASES = {
```

Comment:
Please reduce your amount of test cases a bit as running all source test takes unnecessary longer

Reply:
Reduced